docs-ref-services/latest/messaging-webpubsub-readme.md

```diff
@@ -195,7 +194,7 @@ comments.
 <!-- LINKS -->
 
 [azure_subscription]: https://azure.microsoft.com/free
-[jdk_link]: https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable
+[jdk_link]: /azure/developer/java/fundamental
```

Comment:
this will break the link from github.  can you make it fully qualified?

Reply:
I think the reason it's broken is because of a typo. I missed an 's' on the end fundamentals. I'll push up that change in a minute, and hopefully it'll work.